src/main/java/ChatBro.java

```diff
@@ -0,0 +1,232 @@
+import java.util.ArrayList;
+
+public class ChatBro {
+    public static void main(String[] args) {
+        Ui ui = new Ui();
+        Parser parser = new Parser();
+        Database db = new Database();
+
+        ArrayList<Task> taskList = new ArrayList<>(101);
```

Comment:
Perhaps include a comment explaining why size 101 was chosen would be good for understanding the code?